### PR TITLE
Bluetooth: controller: Fix endianness for length update.

### PR DESCRIPTION
**Pull request by wopu-ot**
Fix endianness when accessing length update parameters in PDU.

Signed-off-by: Wolfgang Puffitsch <wopu@demant.com>

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull_conn.c [hunk 1]
```diff
@@ -3913,25 +3926,28 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 		/* use the minimal of our default_tx_octets and
 		 * peer max_rx_octets
 		 */
-		if (lr->max_rx_octets >= PDU_DC_PAYLOAD_SIZE_MIN) {
-			eff_tx_octets = MIN(lr->max_rx_octets,
+		if (sys_le16_to_cpu(lr->max_rx_octets) >=
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/ull_conn.c]: Use auto-variables to hold in machine endian and use in the if-clauses that check-assign to variables.

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_conn.c (hunk 1 above)
Comment (wopu-ot) [reply subsys/bluetooth/controller/ll_sw/ull_conn.c]: Good point. Done.

**Review by mtpr-ot — APPROVED** [review]
LGTM

**Review by cvinayak — APPROVED** [review]
LGTM

**Pull request merged**